python/src/FunctionalChaosResult.i

```diff
@@ -9,3 +9,94 @@
 %include openturns/FunctionalChaosResult.hxx
 
 namespace OT{ %extend FunctionalChaosResult { FunctionalChaosResult(const FunctionalChaosResult & other) { return new OT::FunctionalChaosResult(other); } } }
+
+
+%pythoncode %{
+def __FunctionalChaosResult_repr_html(self):
+    """Get HTML representation."""
+    html = ""
+    coefficients = self.getCoefficients()
+    basis = self.getOrthogonalBasis()
+    enumerateFunction = basis.getEnumerateFunction()
+    indices = self.getIndices()
+    indicesSize = indices.getSize()
+    # Table of attributes
+    inputDimension = self.getMetaModel().getInputDimension()
+    outputDimension = self.getMetaModel().getOutputDimension()
+    relativeErrors = self.getRelativeErrors()
+    residuals = self.getResiduals()
+    transformation = self.getTransformation()
+    inverseTransformation = self.getInverseTransformation()
+    orthogonalBasis = self.getOrthogonalBasis()
+    distribution = self.getDistribution()
+    fmt = openturns.common.ResourceMap.GetAsString("Sample-PrintFormat")
+    ell_threshold = openturns.common.ResourceMap.GetAsUnsignedInteger("Sample-PrintEllipsisThreshold")
+    ell_size = openturns.common.ResourceMap.GetAsUnsignedInteger("Sample-PrintEllipsisSize")
```

Comment:
the keys should come from FunctionalChaosResult, like you did for FCSensitivity

Reply:
This is true. I must fix that.

Reply:
Fixed at d15c92f80de4e14ead83233fe3a9ff12e1ea8617